.github/workflows/phpunit-tests.yml

```diff
@@ -249,24 +250,31 @@ jobs:
         run: npm run env:install
 
       - name: Run PHPUnit tests
+        if: ${{ ! matrix.multisite }}
         run: npm run test:${{ env.PHPUNIT_SCRIPT }} -- --verbose -c phpunit.xml.dist
 
       - name: Run AJAX tests
+        if: ${{ ! matrix.multisite }}
         run: npm run test:${{ env.PHPUNIT_SCRIPT }} -- --verbose -c phpunit.xml.dist --group ajax
 
       - name: Run tests as a multisite install
+        if: ${{ matrix.multisite }}
         run: npm run test:${{ env.PHPUNIT_SCRIPT }} -- --verbose -c tests/phpunit/multisite.xml
 
       - name: Run AJAX tests as a multisite install
+        if: ${{ matrix.multisite }}
         run: npm run test:${{ env.PHPUNIT_SCRIPT }} -- --verbose -c tests/phpunit/multisite.xml --group ajax
 
       - name: Run ms-files tests as a multisite install
+        if: ${{ matrix.multisite }}
         run: npm run test:${{ env.PHPUNIT_SCRIPT }} -- --verbose -c tests/phpunit/multisite.xml --group ms-files
 
       - name: Run external HTTP tests
+        if: ${{ ! matrix.multisite }}
         run: npm run test:${{ env.PHPUNIT_SCRIPT }} -- --verbose -c phpunit.xml.dist --group external-http
 
       - name: Run REST API tests
+        if: ${{ ! matrix.multisite }}
         run: npm run test:${{ env.PHPUNIT_SCRIPT }} -- --verbose -c phpunit.xml.dist --group restapi-jsclient
 
       # __fakegroup__ is excluded to force PHPUnit to ignore the <exclude> settings in phpunit.xml.dist.
```

Comment:
Do these need to run on multisite? I don't think there is anything multisite specific in these tests, so we could probably just skip them on multisite as well.

Reply:
They don't currently run on Multisite, but they could and technically should. If we introduce REST API endpoints for sites or networks we'd want these tests running on Multisite.

Reply:
Ah, sorry. The line I chose to comment on was misleading in the view above. This was in reference to the xDebug tests, not REST API.

Reply:
Ah sorry. The slowest that these Xdebug tests take is on the PHP 5.6 job at under 15 seconds so I'm happy to leave them in both the single site and Multisite runs.